tools/bin/commands/release

```diff
@@ -235,6 +232,71 @@ update_image_versions() {
     sed "s/TagMajor: .*$/TagMajor: \"$next_version\"/" -i $topdir/install/operator/build/conf/config.yaml
 }
 
+get_github_username() {
+    if [ -z "${GITHUB_USERNAME:-}" ]; then
+        echo "ERROR: environment variable GITHUB_USERNAME has not been set."
+        echo "Please populate it with your github id"
+        return 1
+    fi
+    echo $GITHUB_USERNAME
+}
+
+get_github_access_token() {
+    if [ -z "${GITHUB_ACCESS_TOKEN:-}" ]; then
+        echo "ERROR: environment variable GITHUB_ACCESS_TOKEN has not been set."
+        echo "Please populate it with a valid personal access token from github (with 'repo', 'admin:org_hook' and 'admin:repo_hook' scopes)."
+        return 1
+    fi
+    echo $GITHUB_ACCESS_TOKEN
+}
+
+prepare_binaries() {
+    local from=$1
+    local to=$2
+
+    if ! [[ -d ${from} ]]; then
+        echo "ERROR: The directory where the binaries are located must be a valid directory, got [${from}]"
+        return 1
+    fi
+
+    if ! [[ -d ${to} ]]; then
+        mkdir ${to}
+    fi
+
+    for dist in darwin-amd64 linux-amd64 windows-amd64; do
+        cp ${from}/${dist}/syndesis-operator ${to}/syndesis-operator-${dist}
+    done
+}
+
+publish_artifacts() {
+    local release_dir=$1
+    local tag=$2
+
+    get_github_access_token
+    get_github_username
+
+    set +e
+    local upload_url=$(curl -q --fail -X POST -u $GITHUB_USERNAME:${GITHUB_ACCESS_TOKEN} -H "Accept: application/vnd.github.v3+json" -H "Content-Type: application/json" -d "{\"tag_name\": \"${tag}\"}" https://api.github.com/repos/syndesisio/syndesis/releases | jq -r .upload_url | cut -d{ -f1)
```

Comment:
We could add `prerelease: true` to the payload here if this is a snapshot release.